kura/org.eclipse.kura.linux.watchdog/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,10 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Service-Component: OSGI-INF/*.xml
 Bundle-ClassPath: .
 Bundle-ActivationPolicy: lazy
-Import-Package: org.eclipse.kura.configuration;version="[1.1,1.2)",
+Import-Package: org.eclipse.kura;version="[1.2.0,2.0.0)",
+ org.eclipse.kura.configuration;version="[1.1,1.2)",
```

Comment:
Here we are implementing org.eclipse.kura.configuration.ConfigurableComponent as consumers.
See [Wiki page](https://github.com/eclipse/kura/wiki/Kura-Semantic-Versioning).
Hence we need to import it with [1.1,2.0)